README.md

```diff
@@ -409,6 +409,8 @@ archive format on S3. You can use several format:
 *   gzip_command (Need gzip command)
     *   This compressor uses an external gzip command, hence would result in
         utilizing CPU cores well compared with `gzip`
+*   parquet (Need columnify command)
+    *   This compressor uses an external [columnify](https://github.com/reproio/columnify) command.
```

Comment:
@okkez 
It seems to require more explanation in README.md to use this compressor compared with the other compressors. Otherwise users need to read the source code or find this issue. Could you add it?

Reply:
OK, I'll add some more explanation.

Reply:
Thanks!
Hmm, it's not your fault but the structure of the sentence is a bit hard to understand.
I'll try to refine it in another PR.